src/main/iidxhook9/fs-hook.c

```diff
@@ -0,0 +1,50 @@
+#define LOG_MODULE "fs-hook"
+
+#include <stdint.h>
+#include <string.h>
+
+#include "hook/table.h"
+
+#include "imports/avs.h"
+
+#include "iidxhook9/fs-hook.h"
+
+#include "util/log.h"
+#include "util/str.h"
+
+static void *(*real_avs_fs_mount)(const char *dest, const char *src, const char *fs_type, const char *options);
+static void *my_avs_fs_mount(const char *dest, const char *src, const char *fs_type, const char *options);
+
+static const struct hook_symbol avs_fs_hook_syms[] = {
+    {.name = "XCgsqzn000004b", // avs_fs_mount
+     .ordinal = 76,
+     .patch = my_avs_fs_mount,
+     .link = (void **) &real_avs_fs_mount},
+};
+
+static void *my_avs_fs_mount(const char *dest, const char *src, const char *fs_type, const char *options)
+{
+    // quick check for "F:\"
+    if (src[0] == 'F' && src[1] == ':' && src[2] == '\0') {
+        const char* dev_folder_drive = "dev/vfs/drive_f/";
+        log_misc("Redirecting %s to %s", src, dev_folder_drive);
+
+        CreateDirectoryA("dev/vfs/", NULL);
+        CreateDirectoryA("dev/vfs/drive_f/", NULL);
```

Comment:
suggestion: Not happy with the solution as it forces you to a hardcoded path. Maybe someone still wants these recordings on a separate disk or usb thumb drive? How about making this path configurable?

Reply:
Sounds reasonable, I wanted to avoid a config originally, I just looked up the documentation for CreateDirectoryA, looks like I can just pass the full path in, no need to iterate, will update